src/IssueViz.UnitTests/Editor/LocationTagging/LocationTaggerTests.cs

```diff
@@ -364,6 +364,41 @@ public void GetTags_DifferentSnapshots_SnapshotsAreTranslated()
             testSubject.TagSpans[1].Tag.Location.Span.Value.Snapshot.Should().Be(snapshotMock2);
         }
 
+
+        [TestMethod]
+        public void GetTags_InvalidatedSpans_ShouldBeFiltered()
```

Comment:
A lot of the code in this test is made to assert span translations.
I think it can be simplified, as we don't care about translation in this case, we just want the tag to be removed completely.
`GetTags_HasIntersectingSpans_ReturnsExpectedTags` is a shorter version, what do you think about basing the test on that?